README.md

```diff
@@ -0,0 +1,108 @@
+# Perform
+
+Easy dependency injection for storyboard segues.
+
+```swift
+import Perform
+
+// ...
+
+func tableView(_ tableView: UITableView, didSelectRowAt indexPath: NSIndexPath) {
+  let task = taskList[indexPath.row]
+  perform(.showTaskDetails) {
+    $0.task = task
```

Comment:
Cool, I think the placeholder style is fine for regular usage since you know what's happening, but for the first example in the readme, I think it makes sense to be explicit. 👍